pyproject.toml

```diff
@@ -17,3 +17,5 @@ test-command = "pytest {project}/tests"
 # PyPy builds fail with an error about ninja, probably https://github.com/pypa/setuptools/issues/2912
 # tbb-devel not available for musllinux
 skip = ["pp*", "*musllinux*"]
+build = ["cp39-manylinux_x86_64", "cp38-macosx_x86_64", "cp39-win_amd64"]
```

Comment:
What does the `cp` prefix mean? And why is it different for MacOS?

Reply:
Just for testing, made only one package per OS, this will be removed. 3.8 on mac, because my Laptop has that Python version.